xapi-profiles-structure.md

```diff
@@ -80,7 +80,180 @@ To assist in accomplishing these two primary goals, Profiles also contain metada
+B | Version | `version_b` |`https://example.com/profile/b/version/1.0.0`
+B | Statement Template | `template_z` | `https://example.com/profile/b/templates/z`
+B | Pattern | `pattern_j` | `https://example.com/profile/b/pattern/j`
+
+The following code blocks demonstrate the way Category Context Activities can be used to reference a Profile. The implications of the reference are enumerated below the code blocks.
+
+#### Profile Version
+
+A reference to a Profile Version is the broadest link between a Statement and a Profile.
+
+```javascript
+{
+...
+"context": {"category": [{"id": "https://example.com/profile/a/version/1.0.0"}]}
+...
+}
+
+```
+This Statement MUST:
+* conform to either `template_x` or `template_y`
+
+```javascript
+{
+...
+"context": {"category": [{"id": "https://example.com/profile/a/version/1.0.0"},
+                         {"id": "https://example.com/profile/b/version/1.0.0"}]}
+...
+}
+
+```
+This Statement MUST:
+* conform to either `template_x` or `template_y`
+* conform to `template_z`
+
+#### Statement Template
+
+References to a Statement Template allow for more precise linking between a Statement and Profile.
+
+Because [Statement Templates](./xapi-profiles-structure.md#statment-templates) are highly configurable, its possible that a single Statement conforms to multiple Statement Templates within a Profile or across Profiles.
+
+```javascript
+{
+...
+"context": {"category": [{"id": "https://example.com/profile/a/templates/x"}]}
+...
+}
+```
+
+This Statement MUST:
+* conform to `template_x`
+
+```javascript
+{
+...
+"context": {"category": [{"id": "https://example.com/profile/a/templates/x"},
+                         {"id": "https://example.com/profile/b/templates/z"}]}
+...
+}
+```
+
+This Statement MUST:
+* conform to `template_x`
+* conform to `template_z`
+
+#### Patterns
+
+References to a Pattern allow for linking between a Statement, a Profile and a collection of Statements.
+
+The following assertions are made about the relationship between the example Statement Templates and Patterns. Please see [Patterns](./xapi-profiles-structure.md#patterns) for more information on the types of relationships and corresponding constraints.
+
+* `pattern_i` is a sequence of `template_x`, `template_y`
+* `pattern_j` is a sequence of `template_y`, `template_z`
+
+```javascript
+{
+...
+"context": {"category": [{"id": "https://example.com/profile/a/pattern/i"}]}
+...
+}
+```
+
+This Statement MUST:
+* conform to `template_x` or `template_y`
+
+```javascript
+{
+...
+"context": {"category": [{"id": "https://example.com/profile/a/pattern/i"},
+                         {"id": "https://example.com/profile/b/pattern/j"}]}
+...
+}
+```
+
+This Statement MUST:
+* conform to `template_x` or `template_y`
+* conform to `template_y` or `template_z`
+
+#### Combinations
+
+Combining references to Profiles and Profile Components provides flexible, declarative and precise linking.
```

Comment:
profile is sufficient for validation (assuming always included in stmt that conforms to profile)

not so much combinations but specificity, validation checks against overall template and then template / pattern references are metadata that can be utilized by xAPI Profile aware systems

-> pre-defined / reserved within xAPI Profile produced within standardization effort. Correct usage concern when testing. Do not place unnecessary restrictions on them, they are free to use like any other concept defined in a profile